python_scripts/parse_kraken_to_biom.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+
+#-----------------------------------------------------------------------------
+# Copyright (c) 2016--, Evguenia Kopylova.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+"""
+Parse output of "kraken translate" and generate single BIOM table.
+"""
+
+import click
+import pandas as pd
+import numpy as np
+from biom.table import Table
+from biom.util import biom_open, HAVE_H5PY
+
+
+def compute_biom_table(kraken_translate_report_fp,
+                       taxonomic_rank,
+                       taxa_levels,
+                       taxa_levels_idx,
+                       columns,
+                       index):
+    """Absolute abundance of number of reads matching a defined taxa level.
+
+    Parameters
+    ----------
+    kraken_translate_report_fp: string
```

Comment:
string -> str

wont comment more on these, but the expectation is to use the type (although its not actually enforced anywhere that I know of)